src/hotspot/share/compiler/compileLog.cpp

```diff
@@ -113,7 +113,7 @@ int CompileLog::identify(ciBaseObject* obj) {
       if (!klass->is_loaded()) {
         print(" unloaded='1'");
       } else {
-        print(" flags='%d'", klass->modifier_flags());
+        print(" flags='%d'", klass->access_flags());
```

Comment:
There may be tools that parse the log file and get confused by this change.  Maybe we should also change the label from "flags" to "access flags".